packages/docusaurus-theme-classic/src/theme/DocPage/index.tsx

```diff
@@ -5,21 +5,21 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, {type ReactNode, useState, useCallback} from 'react';
-import {MDXProvider} from '@mdx-js/react';
+import React, { type ReactNode, useState, useCallback } from 'react';
```

Comment:
This is weird. Does your editor's formatter not respect the `.prettierrc` file?

Reply:
Generally yes, but it appears not to have in this instance. It may be the dev containers interfering. I'll reset that back :-)